build-release-pkgs.xml

```diff
@@ -115,7 +115,13 @@
     <zip basedir="${workdir}" destfile="${workdir}/narayana-full-${tag}-src.zip" includes="${tag}/**"/>
     <!-- build the binary releases -->
     <exec dir="${workdir}/${tag}" executable="${mvn.executable}" failonerror="true">
-      <arg line="deploy -DskipTests -gs tools/maven/conf/settings.xml -Dorson.jar.location=${user.dir}/ext/ -Prelease"/>
+      <arg line="clean deploy -DskipTests -gs tools/maven/conf/settings.xml -Dorson.jar.location=${user.dir}/ext/ -Prelease"/>
+    </exec>
+    <exec dir="${workdir}/${tag}" executable="${mvn.executable}" failonerror="true">
+      <arg line="clean deploy -DskipTests -gs tools/maven/conf/settings.xml -Prelease -f blacktie/utils/cpp-plugin/pom.xml"/>
+    </exec>
+    <exec dir="${workdir}/${tag}" executable="${mvn.executable}" failonerror="true">
+      <arg line="clean deploy -DskipTests -gs tools/maven/conf/settings.xml -Prelease  -f blacktie/pom.xml -pl :blacktie-jatmibroker-nbf -am"/>
```

Comment:
cpp-plugin lives outside the lifecycle of the main build - it should really be in a separate repo tbh but it was a decision we made a while back. It basically is a plugin used by the main build to build the C++ components under maven in a platform neutral manner. You can't actually call it directly from the maven build as it is cycling during maven initialization.